ddpui/core/orgdbt_manager.py

```
@@ -112,7 +112,7 @@ def run_dbt_command(
 
             cmd = [params.dbt_binary] + command
 
-            result = subprocess.run(cmd, cwd=params.project_dir, capture_output=True, text=True)
+            result = subprocess.run(cmd, cwd=params.org_project_dir, capture_output=True, text=True)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)